src/app/state/rest/index.ts

```diff
@@ -17,6 +17,7 @@ export type ApiGroup =
   | "auth"
   | "supportContacts"
   | "teams"
+  | "reasons"
```

Comment:
What is the purpose of these apiGroups? (just checking)

Reply:
ApiGroups allow for clearing the group's cache on a mutable method call (POST, PUT, PATCH, DELETE). So when you update `"reasons"` for example, it will trigger a new GET request refetching all reasons.